### PR TITLE
Baselining smaller changes in transformation-based emit

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
@rbuckton

### CODE REVIEW

File: tests/baselines/reference/numericClassMembers1.js [hunk 1]
```diff
@@ -19,21 +19,21 @@ class C236 {
 var C234 = (function () {
     function C234() {
         this[0] = 1;
-        this[0.0] = 2;
+        this[0] = 2;
```
Comment (rbuckton) [inline tests/baselines/reference/numericClassMembers1.js]: This is an error, it should be using the raw literal text. I have a similar issue with binary/octal integers I need to investigate.

File: tests/baselines/reference/outModuleConcatCommonjs.js [hunk 2]
```diff
@@ -17,6 +17,23 @@ var __extends = (this && this.__extends) || function (d, b) {
...
+"use strict";
+var a_1 = require("./ref/a");
+var B = (function (_super) {
+    __extends(B, _super);
+    function B() {
+        _super.apply(this, arguments);
+    }
+    return B;
+}(a_1.A));
+exports.B = B;
```
Comment (rbuckton) [inline tests/baselines/reference/outModuleConcatCommonjs.js]: I'm wary of accepting any new baseline for a test that specifies `outFile`

File: tests/baselines/reference/outModuleConcatCommonjs.sourcemap.txt [hunk 3]
```diff
@@ -16,11 +16,94 @@ sourceFile:tests/cases/compiler/ref/a.ts
 >>>// This should be an error
 1 >
 2 >^^^^^^^^^^^^^^^^^^^^^^^^^^
-3 >                          ^^^^->
```
Comment (rbuckton) [inline tests/baselines/reference/outModuleConcatCommonjs.sourcemap.txt]: As above regarding `outFile`, I would rather we didn't take this baseline. Also, I'd be wary of accepting any baseline for source maps as well.

File: tests/baselines/reference/outModuleConcatES6.js [hunk 4]
```diff
@@ -12,6 +12,11 @@ export class B extends A { }
 
 //// [all.js]
 // This should be an error
+export class A {
```
Comment (rbuckton) [inline tests/baselines/reference/outModuleConcatES6.js]: This is another case where I'd rather we not accept this file as it uses `outFile`.

File: tests/baselines/reference/outModuleConcatES6.sourcemap.txt [hunk 5]
```diff
@@ -11,11 +11,89 @@ sourceFile:tests/cases/compiler/ref/a.ts
...
 1 >
 2 >^^^^^^^^^^^^^^^^^^^^^^^^^^
-3 >                          ^^^^->
 1 >
   >
 2 >// This should be an error
 1 >Emitted(1, 1) Source(2, 1) + SourceIndex(0)
 2 >Emitted(1, 27) Source(2, 27) + SourceIndex(0)
 ---
+>>>export class A {
```
Comment (rbuckton) [inline tests/baselines/reference/outModuleConcatES6.sourcemap.txt]: This is another case where I'd rather we not accept this file as it uses `outFile`.

File: tests/baselines/reference/outModuleConcatCommonjs.js.map [hunk 6]
```diff
@@ -1,2 +1,2 @@
 //// [all.js.map]
-{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":";;;;;AACA,0BAA0B"}
\ No newline at end of file
+{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":";;;;;AACA,0BAA0B;;AAE1B;;;;KAAkB;AAAlB,YAAa,CAAC,CAAI;;ACHlB,6BAA0B;AAC1B;;;;;;EAAuB,KAAC,GAAI;AAA5B,YAAa,CAAC,CAAc"}
```
Comment (rbuckton) [inline tests/baselines/reference/outModuleConcatCommonjs.js.map]: This is another case where I'd rather we not accept this file as it uses `outFile`.

File: tests/baselines/reference/outModuleConcatES6.js.map [hunk 7]
```diff
@@ -1,2 +1,2 @@
 //// [all.js.map]
-{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":"AACA,0BAA0B"}
\ No newline at end of file
+{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":"AACA,0BAA0B;AAE1B,MAAM,OAAO,CAAC;CAAI;ACHlB,OAAO,EAAC,CAAC,EAAC,MAAM,SAAS,CAAC;AAC1B,MAAM,OAAO,CAAE,SAAQ,CAAC;CAAI"}
```
Comment (rbuckton) [inline tests/baselines/reference/outModuleConcatES6.js.map]: This is another case where I'd rather we not accept this file as it uses `outFile`.

File: tests/baselines/reference/outModuleConcatUmd.js [hunk 8]
```diff
@@ -17,6 +17,41 @@ var __extends = (this && this.__extends) || function (d, b) {
     d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
 };
 // This should error
+(function (dependencies, factory) {
```
Comment (rbuckton) [inline tests/baselines/reference/outModuleConcatUmd.js]: This is another case where I'd rather we not accept this file as it uses `outFile`.

File: tests/baselines/reference/outModuleConcatUmd.js.map [hunk 9]
```diff
@@ -1,2 +1,2 @@
 //// [all.js.map]
-{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":";;;;;AACA,oBAAoB"}
\ No newline at end of file
+{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":";;;;;AACA,oBAAoB;;;;;;;;;;IAEpB;;;;SAAkB;IAAlB,YAAa,CAAC,CAAI;;;;;;;;;;;ICHlB,6BAA0B;IAC1B;;;;;;MAAuB,KAAC,GAAI;IAA5B,YAAa,CAAC,CAAc"}
```
Comment (rbuckton) [inline tests/baselines/reference/outModuleConcatUmd.js.map]: This is another case where I'd rather we not accept this file as it uses `outFile`.

File: tests/baselines/reference/outModuleConcatUmd.sourcemap.txt [hunk 10]
```diff
@@ -16,11 +16,112 @@ sourceFile:tests/cases/compiler/ref/a.ts
 >>>// This should error
```
Comment (rbuckton) [inline tests/baselines/reference/outModuleConcatUmd.sourcemap.txt]: This is another case where I'd rather we not accept this file as it uses `outFile`.

**Comment by mhegazy** [comment]
@DanielRosenwasser do not think this is needed any longer. if this is not true, please update.

**Pull request closed without merging**